src/meinberlin/meinberlin/static/icons/preview/adhocracy3-icons-preview.html

```diff
@@ -920,19 +903,6 @@
       
       <div class="glyph">
         <div class="preview-glyphs">
-          <span class="step size-12"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-14"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-16"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-18"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-21"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-24"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-36"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-48"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-60"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span><span class="step size-72"><span class="letters">Pp</span><i id="icon-voting-gap" class="icon-voting-gap"></i></span>
-        </div>
-        <div class="preview-scale">
-          <span class="step">12</span><span class="step">14</span><span class="step">16</span><span class="step">18</span><span class="step">21</span><span class="step">24</span><span class="step">36</span><span class="step">48</span><span class="step">60</span><span class="step">72</span>
-        </div>
-        <div class="usage">
-          <input class="class" type="text" readonly="readonly" onClick="this.select();" value=".icon-voting-gap" />
-          <input class="point" type="text" readonly="readonly" onClick="this.select();" value="&amp;#xf130;" />
-        </div>
-      </div>
-      
-      <div class="glyph">
-        <div class="preview-glyphs">
```

Comment:
This file is automatically generated by [font custom](https://fontcustom.github.io/fontcustom/), so you do not need to edit it by hand. But it also doesn't hurt.